com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -4,16 +4,18 @@
 using MLAPI.Serialization.Pooled;
 using MLAPI.Profiling;
 using MLAPI.Transports;
+using MLAPI.Logging;
 
 namespace MLAPI.Messaging
 {
     /// <summary>
     /// RpcQueueContainer
     /// Handles the management of an Rpc Queue
     /// </summary>
-    internal class RpcQueueContainer : INetworkUpdateSystem
+    internal class RpcQueueContainer : INetworkUpdateSystem, IDisposable
     {
         private const int k_MinQueueHistory = 2; //We need a minimum of 2 queue history buffers in order to properly handle looping back Rpcs when a host
+        private static int s_RpcQueueContainerInstances;
```

Comment:
Let's surround this with #if UNITY_EDITOR || DEVELOPMENT_BUILD as it was before.  It communicates to the reader that this variable only means anything when in that state, which I had to figure out by searching through the code

Reply:
Hahah, yeah already done!